app/views/content/landing/if-you-have-a-degree.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Train to be a teacher if you have a degree"
```

Comment:
this looks slightly overbearing on so many lines @EllieNodder !

Would it work to just call it 'Train to be a teacher' or 'Train to teach'?

It is only a landing page (so we are directing traffic to it) so h1 less important for search. It is also not a category card page on Train to be a teacher so doesn't need to have different h1s/titles from that perspective?